<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client_Window</class>
 <widget class="QMainWindow" name="Client_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>538</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Client Window</string>
  </property>
  <property name="windowIcon">
   <iconset resource="client.qrc">
    <normaloff>:/images/app_icon_s.png</normaloff>:/images/app_icon_s.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="chat_text">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>641</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="chat_input">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>460</y>
      <width>701</width>
      <height>30</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Write your messages here</string>
    </property>
   </widget>
   <widget class="QPushButton" name="send_button">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>460</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QListWidget" name="chat_active">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>131</width>
      <height>491</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::SelectedClicked</set>
    </property>
   </widget>
   <widget class="QListWidget" name="chat_nicks">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>0</y>
      <width>141</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="smiles_button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>460</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="client.qrc">
      <normaloff>:/images/smile_00.png</normaloff>:/images/smile_00.png</iconset>
    </property>
   </widget>
   <widget class="QWidget" name="chat_smilies" native="true">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>310</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="smile_00">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="client.qrc">
       <normaloff>:/images/smile_00.png</normaloff>:/images/smile_00.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="smile_01">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="client.qrc">
       <normaloff>:/images/smile_01.png</normaloff>:/images/smile_01.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="smile_02">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>0</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="client.qrc">
       <normaloff>:/images/smile_02.png</normaloff>:/images/smile_02.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="smile_03">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>0</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="client.qrc">
       <normaloff>:/images/smile_03.png</normaloff>:/images/smile_03.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="smile_04">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>0</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="client.qrc">
       <normaloff>:/images/smile_04.png</normaloff>:/images/smile_04.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="smile_12">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="client.qrc">
       <normaloff>:/images/smile_12.png</normaloff>:/images/smile_12.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="smile_14">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="client.qrc">
       <normaloff>:/images/smile_14.png</normaloff>:/images/smile_14.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="smile_13">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="client.qrc">
       <normaloff>:/images/smile_13.png</normaloff>:/images/smile_13.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="smile_11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="client.qrc">
       <normaloff>:/images/smile_11.png</normaloff>:/images/smile_11.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="smile_10">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="client.qrc">
       <normaloff>:/images/smile_10.png</normaloff>:/images/smile_10.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="smile_22">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="client.qrc">
       <normaloff>:/images/smile_22.png</normaloff>:/images/smile_22.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="smile_24">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="client.qrc">
       <normaloff>:/images/smile_24.png</normaloff>:/images/smile_24.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="smile_23">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="client.qrc">
       <normaloff>:/images/smile_23.png</normaloff>:/images/smile_23.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="smile_21">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="client.qrc">
       <normaloff>:/images/smile_21.png</normaloff>:/images/smile_21.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="smile_20">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="client.qrc">
       <normaloff>:/images/smile_20.png</normaloff>:/images/smile_20.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="smile_32">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="client.qrc">
       <normaloff>:/images/smile_32.png</normaloff>:/images/smile_32.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="smile_34">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="client.qrc">
       <normaloff>:/images/smile_34.png</normaloff>:/images/smile_34.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="smile_33">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="client.qrc">
       <normaloff>:/images/smile_33.png</normaloff>:/images/smile_33.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="smile_31">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="client.qrc">
       <normaloff>:/images/smile_31.png</normaloff>:/images/smile_31.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="smile_30">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="client.qrc">
       <normaloff>:/images/smile_30.png</normaloff>:/images/smile_30.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="smile_42">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="client.qrc">
       <normaloff>:/images/smile_42.png</normaloff>:/images/smile_42.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="smile_44">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="client.qrc">
       <normaloff>:/images/smile_44.png</normaloff>:/images/smile_44.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="smile_43">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="client.qrc">
       <normaloff>:/images/smile_43.png</normaloff>:/images/smile_43.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="smile_41">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="client.qrc">
       <normaloff>:/images/smile_41.png</normaloff>:/images/smile_41.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="smile_40">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="client.qrc">
       <normaloff>:/images/smile_40.png</normaloff>:/images/smile_40.png</iconset>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>948</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuClient">
    <property name="title">
     <string>Server</string>
    </property>
    <addaction name="actionNew_serv"/>
    <addaction name="actionClose_serv"/>
    <addaction name="separator"/>
    <addaction name="actionJoin_chan"/>
    <addaction name="actionPart_chan"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuProfile">
    <property name="title">
     <string>Profile</string>
    </property>
    <addaction name="actionNew_acc"/>
    <addaction name="actionChange_acc"/>
    <addaction name="actionDelete_acc"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuClient"/>
   <addaction name="menuProfile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_serv">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionClose_serv">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionNew_acc">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionChange_acc">
   <property name="text">
    <string>Change</string>
   </property>
  </action>
  <action name="actionDelete_acc">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionJoin_chan">
   <property name="text">
    <string>Join channel</string>
   </property>
  </action>
  <action name="actionPart_chan">
   <property name="text">
    <string>Part channel</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="client.qrc"/>
 </resources>
 <connections/>
</ui>
